src/gluon/events/project/ProjectProductionEnvironmentsRequested.ts

```diff
@@ -56,7 +55,7 @@ export class ProjectProductionEnvironmentsRequested implements HandleEvent<any>
 
             for (const teamMember of membersToMessage) {
                 const requestCorrelationId: string = uuid();
-                const destination = await addressSlackUsersFromContext(ctx, teamMember.slack.screenName);
+                const destination =  await addressSlackUsersFromContext(ctx, teamMember.slack.screenName);
```

Comment:
Remove the extra space
```suggestion
                const destination = await addressSlackUsersFromContext(ctx, teamMember.slack.screenName);
```